src/Library/demos/GTK Switch/main.blp

```diff
@@ -0,0 +1,55 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "GTK Switch";
+
+  Box {
+    orientation: vertical;
+
+     Label {
+        label: _("GtkSwitch is a “light switch” that has two states: on or off.\nThe user can control which state should be active by clicking the empty area, or by dragging the handle.\n");
+        justify: center;
+        margin-bottom: 15;
+        styles ["body"]
+      }
+
+    Box {
+      orientation: vertical;
+      halign: center;
+
+      Switch one{
+      margin-bottom: 10;
+      active: true;
+      }
+    }
+    Label switch_one{
+    label: "Currently in Active state";
+    margin-bottom: 40;
+    }
+
+    Box {
+     orientation: vertical;
+     halign: center;
+
+     Switch second{
+     margin-bottom: 10;
+     }
+    }
+
+    Label switch_second{
+    label: "Currently in Inactive state";
+    margin-bottom: 20;
+    }
+
+    Label {
+    label: "Try turning the switches on/off.";
+    margin-bottom: 10;
+    }
+
+    LinkButton{
+    label: "Find More Info here";
+    uri: "https://docs.gtk.org/gtk3/class.Switch.html";
+    }
```

Comment:
please add a link to https://developer.gnome.org/hig/patterns/controls/switches.html (use the same label as other HIG links)

and an other one to https://developer.gnome.org/documentation/tutorials/beginners/components/switch.html